src/screens/HyperSwitch/Utils/CountryUtils.res

```diff
@@ -751,259 +751,6 @@ type countiesCode =
   | ZM
   | ZW
 
-let getCountryFromCountryCode = code => {
```

Comment:
Can you please revert these changes of CountryUtils as they might be getting used.

Reply:
Done